src/context/AppProvider/AppContext.tsx

```diff
@@ -251,20 +251,19 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     // Exclude assets for which we are unable to get market data
     // We would fire query thunks / XHRs that would slow down the app
     // We insert price data for these as resolver-level, and are unable to get market data
-    const excluded: AssetId[] = [...(uniV2LpIdsData.data ?? [])]
+    const excluded: AssetId[] = uniV2LpIdsData.data ?? []
```

Comment:
didn't need to spread here and this is a decently bigly array

Reply:
Abso-lutely, ty ser